src/picknik_ur_gazebo_config/launch/sim/hardware_sim.launch.py

```diff
@@ -33,23 +33,37 @@
 
 from ament_index_python import get_package_share_directory
```

Comment:
I might push back on the changes to this file. When moving this to this repo, I took care to remove certain things that may not have been needed, so I'll fill in more details in more comments.

Reply:
Understood!

I reverted all but the critical and minor functional changes.